server/src/main/java/org/elasticsearch/common/util/concurrent/ThreadContext.java

```diff
@@ -751,4 +770,40 @@ public AbstractRunnable unwrap() {
             return in;
         }
     }
+
+    private static Collector<String, Set<String>, Set<String>> LINKED_HASH_SET_COLLECTOR = new LinkedHashSetCollector<>();
```

Comment:
This can be declared `final`?

Reply:
I pushed 6e1ecec8f4361e8748e40d04e52b49f41e983d56.